File: php/delivery-api/GetTypes.php
<?php
// DocSection: delivery_api_get_types
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kontent\Ai\Delivery\DeliveryClient;

$client = new DeliveryClient('<YOUR_PROJECT_ID>');

$types = $client->getTypes((new QueryParams())
            ->limit(3));
// EndDocSection
?>

File: ruby/delivery-api/GetTypes.rb
# DocSection: delivery_api_get_types
require 'delivery-sdk-ruby'

delivery_client = Kentico::Kontent::Delivery::DeliveryClient.new project_id: '975bf280-fd91-488c-994c-2f04416e5ee3'
delivery_client.types
               .limit(3)
               .execute do |response|
                 types = response.types
               end
# EndDocSection